include/mlir/Dialect/StandardOps/Ops.td

```diff
@@ -1532,6 +1532,24 @@ def TruncateIOp : Std_Op<"trunci", [NoSideEffect, SameOperandsAndResultShape]> {
   }];
 }
 
+def UIToFPOp : CastOp<"uitofp">, Arguments<(ins AnyType:$in)> {
```

Comment:
We should constrain the input to be an integer type, and the output to a floating point.